app/blueprints/round/services.py

```diff
@@ -185,7 +186,8 @@ def update_existing_round(round_obj, form, user="dummy_user"):
     round_obj.eligibility_config = {"has_eligibility": form.eligibility_config.data}
     round_obj.eoi_decision_schema = {
         "en": convert_form_data_to_json(form.eoi_decision_schema_en.data),
-        "cy": convert_form_data_to_json(form.eoi_decision_schema_cy.data),
+        "cy": convert_form_data_to_json(
+            form.eoi_decision_schema_cy.data) if form.eoi_decision_schema_cy.data and form.eoi_decision_schema_cy.data.lower() != "none" else None,
```

Comment:
`form.eoi_decision_schema_cy.data.lower() != "none"`

does this really happen? 👀